roles/system_packages/vars/main.yml

```diff
@@ -1,88 +1,104 @@
 ---
 pkgs:
```

Comment:
That's the idea. But no need to put everything under pkgs, we should instead add a new top level var IMP (`pkgs_to_remove` for instance ?)

Putting the vars together does not gain us much (the task is not that much more readable), and having two separates vars allows:
- manual overrides with --extra-vars is easier
- diff is more review friendly (here the whole pkgs definition is moved )